### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Chainski**
- [x] Updated ANTI-VM checks

### CODE REVIEW

File: frontend-src/antivm.ps1 [hunk 1]
```diff
@@ -124,12 +128,12 @@ function VMPROTECT {
...
     )
     $detectedProcesses = Get-Process -Name $processnames -ErrorAction SilentlyContinue | Select-Object -ExpandProperty Name
-    if ($null -ne $detectedProcesses) {
+    if ($detectedProcesses -ne $null) {
         Write-Output "Detected processes: $($detectedProcesses -join ', ')"
         Stop-Process $pid -Force
     }
 
-    if ($null -eq $detectedProcesses) {
+    if ($detectedProcesses -eq $null) {
```
Comment (KDot227) [inline frontend-src/antivm.ps1:136]: Have it do the null first or else the powershell style checker screams at me.

**Review by KDot227 — CHANGES_REQUESTED** [review]

File: frontend-src/antivm.ps1 (hunk 1 above)
Comment (Chainski) [reply frontend-src/antivm.ps1:136]: Ok I wondered why you had it backwards. give me a sec

Comment (Chainski) [reply frontend-src/antivm.ps1]: Done !

**Review by Chainski — COMMENTED** [review]
Changes made

**Pull request merged**